python/lsst/ts/genericcamera/exposure.py

```diff
@@ -89,6 +89,9 @@ def make_fileobj(self):
                     hdr_primary.append(item(), end=True)
                 for item in self.header["IMAGE1"]:
                     hdr_image1.append(item(), end=True)
+                    # Updates on image extension keywords since values are not
+                    # broadcast to the GCHeaderService
+                    hdr_image1["DETSIZE"] = f"[1:{self.height},1:{self.width}]"
```

Comment:
Is the indentation correct for this line (and the associated comment lines)? It is inside the for loop now.

Reply:
Oops. That's wrong. I'll fix it.

Reply:
Fixed.